### PR TITLE
add forensics command

### PR DESCRIPTION
**Pull request by jochman**
# Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/18762

## Description
Tested with customer.

## Does it break backward compatibility?
   - No

## Must have
- [X] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](link)
- [ ] [CHANGELOG](link)

### CODE REVIEW

File: Integrations/ProofpointTAP_v2/ProofpointTAP_v2_test.py [hunk 1]
```diff
@@ -312,3 +312,417 @@ def test_get_fetch_times():
...
+        'id': 'string',
+        'forensics': EVIDENCE_LIST,
+    }]
+
+    REPORT = {
+        'generated': 'string',
+        'reports': REPORT_OBJECT * 2,
+    }
+
+    EXPECTED_OUTPUT_REPORT = {
```
Comment (anara123) [inline Integrations/ProofpointTAP_v2/ProofpointTAP_v2_test.py]: you dont call your mock data which is global, EXPECTED*.
You should call it FORENSICS_REPORT for example and in the test assign to local variable
`expected_report = FORENSICS_REPORT`

File: Integrations/ProofpointTAP_v2/ProofpointTAP_v2.py [hunk 2]
```diff
@@ -58,14 +58,19 @@ def get_fetch_times(last_fetch):
 
 class Client:
     def __init__(self, proofpoint_url, api_version, verify, service_principal, secret, proxies):
-        self.base_url = "{}/{}/siem".format(proofpoint_url, api_version)
+        self.base_url = urljoin(proofpoint_url, api_version)
```
Comment (anara123) [inline Integrations/ProofpointTAP_v2/ProofpointTAP_v2.py]: no need for extra `base_url_no_api_version `
just keep the base as it is and save the `self.api_verison = api_version`.
in the http_request pass `forensics_api` of type boolean
```python
if forensics_api:
    full_url = urljoin(self.base_url, '/v2/forensics')
else:
    full_url = urljoin(urljoin(self.base_url, self.api_version), url_suffix)
```

**Comment by Itay4** [comment]
@jochman please resolve conflicts

**Comment by Itay4** [comment]
@richardbluestone please review yml and changelog

**Comment by richardbluestone** [comment]
Reviewed and updated the yml and md files.

**Comment by richardbluestone** [comment]
Can you generate the documentation for the new commands/outputs, so I can add it to Zendesk?

**Pull request merged**